Assets/Scripts/ServiceLocator.cs

```diff
@@ -1,5 +1,6 @@
 ﻿namespace HarmonyQuest
 {
+    using HarmonyQuest.Input.Implementation;

```

Comment:
I believe I already fixed this in a previous PR. Should merge fine, but just a heads up.